src/index.html

```diff
@@ -1,13 +1,67 @@
 <!doctype html>
-<html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+<html lang="en" class="page">
+  <head>
+    <meta charset="UTF-8">
+    <meta 
+      name="viewport"
+      content="
+        width=device-width,
+        user-scalable=no, 
+        initial-scale=1.0, 
+        maximum-scale=1.0, 
+        minimum-scale=1.0
+      "
+    >
+    
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./styles/index.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link 
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" 
+      rel="stylesheet"
+    >
+  </head>
+  <body class="page__container">
+    <div class="card" data-qa="card">
+      <img
+        class="card__iMac-image"
+        src="images/imac.jpeg"
+        alt="iMac"
+      >
+
+      <h2 class="card__name ">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</h2>
+      
+      <div class="card__code">Код товара: 195434</div>
```

Comment:
Follow this code style
```suggestion
      <h2 class="card__name ">
      	Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
      </h2>
      
      <div class="card__code">
      	Код товара: 195434
      </div>
```